src/components/CheckoutForm/CheckoutForm.tsx

```diff
@@ -47,16 +51,17 @@ class CheckoutForm extends React.Component<Props, State> {
     this.handleChange = this.handleChange.bind(this);
     this.handleValueChange = this.handleValueChange.bind(this);
     this.handleSubmit = this.handleSubmit.bind(this);
+    // this.InjectedCheckoutForm = this.InjectedCheckoutForm.bind(this);
   }
 
-  getCheckoutSession() {
+  async getCheckoutSession() {
     const lineItems = this.state.lineItems;
     lineItems.description = `${this.state.formattedValue} ${this.props.option.toLowerCase()} for ${this.props.merchant}`;
     lineItems.amount = lineItems.amount * 100;
-    lineItems.quantity = 1; 
+    lineItems.quantity = 1;
 
     // should return stripe checkout session id once the endpoint is working
-    return axios.post('https://api.sendchinatownlove.com/charges', {
+    const res = await axios.post('http://localhost:3001/charges', {
```

Comment:
We should update this to be localhost when developing, but when we use yarn deploy, it replaces it with api.sendchinatownlove.com